src/internal/connector/graph_connector_helper_test.go

```diff
@@ -1030,7 +1031,7 @@ func collectionsForInfo(
 			baseExpected[info.items[i].lookupKey] = info.items[i].data
 
 			// We do not count metadata files against item count
-			if service != path.OneDriveService ||
+			if countMeta || service != path.OneDriveService ||
```

Comment:
nit: The name `countMeta` seems a bit misleading. How about we just pass in the version number of the backup and use that here?

Reply:
yea, lets try that for now. We may need to do something else in the future if we start getting a lot of different versions we need to switch behavior on, but I guess we'll figure out when we get there ;)